api/app/src/mappings/fitbit/body.ts

```diff
@@ -47,10 +51,15 @@ export const mapToBody = (
         value: weight.weight,
         data_source: {
           name: weight.source,
+          standardized_type: checkSource(weight.source),
         },
       };
     });
   }
 
   return body;
 };
+
+function checkSource(weight: string) {
+  return weight === "API" ? "MANUALLY_ENTERED" : "DEVICE";
```

Comment:
all of these strings can be consts, and `DEVICE` can be re-used in the withings provider